lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -64,7 +64,7 @@ LocalEnforcer::LocalEnforcer(
     std::shared_ptr<aaa::AAAClient> aaa_client,
```

Comment:
This is just my personal preference, but I think we should consolidate the object parameters. 
`std::shared_ptr<ShardTracker> shards` can go below `aaa_client`

Reply:
I stuck the parameters in the end to make debugging easier especially when local enforcer objects are initialized in test files. This also makes sense though.